app/src/main/java/io/appium/settings/RecorderConstant.java

```diff
@@ -0,0 +1,14 @@
+package io.appium.settings;
+
+public class RecorderConstant {
+    public static final String DEFAULT_RECORDING_FILENAME = "AppiumScreenRecord";
+    public static final float BITRATE_MULTIPLIER = 0.25f;
+    public static final int AUDIO_CODEC_SAMPLE_RATE = 44100;
+    public static final int AUDIO_CODEC_CHANNEL_COUNT = 1;
+    public static final int AUDIO_CODEC_REPEAT_PREV_FRAME_AFTER = 1000000;
+    public static final int AUDIO_CODEC_I_FRAME_INTERVAL = 5;
```

Comment:
I usually add unit names to time constants were possible, for example `AUDIO_CODEC_I_FRAME_INTERVAL  -> AUDIO_CODEC_I_FRAME_INTERVAL_MS`

Reply:
done